stable/postgresql/values.yaml

```diff
@@ -210,7 +210,7 @@ extraEnv: []
 # initdbScriptsSecret:
 
 ## Specify the PostgreSQL username and password to execute the initdb scripts
-# initdbUser:
+# initdbUsername:
```

Comment:
please, could you update `stable/postgresql/values-production.yaml` too?

Reply:
done